src/sc/proofverifier.cpp

```diff
@@ -77,7 +77,7 @@ bool CScProofVerifier::BatchVerify() const
     }
 
     CctpErrorCode code;
-    ZendooBatchProofVerifier batchVerifier;
+    ZendooBatchProofVerifier batchVerifier(true);
```

Comment:
what is the meaning of constructor arg equals `true`?
Do we need to change the creation policy in other places, like in `asyncproofverifier::BatchVerify`?

Reply:
The paramenter in the ctor is a boolean telling mc-crypto lib if the rust verifier executing thread will be a high priority one. We can probably add a comment here or a static const var with a speaking name.

Reply:
Ok, I got it. Would be nice to add comment or const.

Reply:
Done

Reply:
It seems that both the CscAsyncProofVerifier and CScProofVerifier instantiate the threads with high priority

Reply:
Right. We must add one more constructor argument to `CScProofVerifier` -> `prioritize` and set it `true` for connect block, `false` for mempool verification.
Moreover, `CScAsyncProofVerifier` should have `prioritize = false` as it is used only for mempool txs/certs.
@albertog78 priority test is present in mc-cryptolib `zendoo_batch_verifier_multiple_threads_with_priority`